cli/main.js

```diff
@@ -14,6 +14,7 @@ const _colors = require('colors');
 const pjson = require('./package.json');
 const { openStdin } = require("process");
 const CLI = true;
+const resourcesArn = []
```

Comment:
maybe change to `physicalIdFilter` or something like that

Reply:
done